app/grandchallenge/github/models.py

```diff
@@ -121,14 +120,29 @@ def user_error(self):
         else:
             return ""
 
+    @property
+    def license(self):
+        # Return the first license found in the results
+        return self.license_check_result.get("licenses", [{}])[0]
```

Comment:
I don't know if this is super robust, we need to run it on a bunch of repos and find out.

Reply:
Or, maybe check _all_ of the licenses found.

Reply:
Maybe check all licenses and as long as there's an open source license, the check passes?

Reply:
Though I'd say multi-licensing is an edge case that might not be worth the added complexity.

Reply:
Exactly, as this is dynamic we can always change it later. Lets see what data we get.